server/hf-kernels.lock

```diff
@@ -0,0 +1,6740 @@
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=1792,device_name=NVIDIA_A100-SXM4-40GB.json",
+        "blob_id": "f4c0f8417b384870050a95e0cf57edbdf6352b23"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=1792,device_name=NVIDIA_A100-SXM4-80GB.json",
+        "blob_id": "5c8185cfdeec167ec4b88de51b4b395e28769cc5"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=1792,device_name=NVIDIA_H100_80GB_HBM3.json",
+        "blob_id": "97c9f4445b166657ad29f1db9fc8281f9c463ec4"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=2048,device_name=NVIDIA_A100-SXM4-80GB.json",
+        "blob_id": "0bb423b28f5ab3825929a4870b96393262a9dd9f"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=2048,device_name=NVIDIA_H100_80GB_HBM3,dtype=fp8_w8a8.json",
+        "blob_id": "55571873395464a3b58f549523905f439a8f1716"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=2048,device_name=NVIDIA_H100_80GB_HBM3.json",
+        "blob_id": "26bcbf26970c7a77c99e2c8eacd83eefa86967bf"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=3584,device_name=AMD_Instinct_MI300X.json",
+        "blob_id": "91011e64c7de4505e9bb462bc70e6a3e7affa878"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=3584,device_name=NVIDIA_A100-SXM4-40GB.json",
+        "blob_id": "b41f9d443e50678334f906b44fce6d018d69500e"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=3584,device_name=NVIDIA_A100-SXM4-80GB.json",
+        "blob_id": "edf2a38d12ad3f420f232d2cd61ab149ad138725"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=3584,device_name=NVIDIA_H100_80GB_HBM3,dtype=fp8_w8a8.json",
+        "blob_id": "673bae2ba8ef80ed4d4930739ca7daf0e8f28ee1"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=3584,device_name=NVIDIA_H100_80GB_HBM3.json",
+        "blob_id": "b2100cebb7f589747430be9ca8c8db368c152d78"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=3584,device_name=NVIDIA_L40S.json",
+        "blob_id": "d720deb4bdd73d194b1023c99e190b8fcfecdaef"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=4096,device_name=NVIDIA_A100-SXM4-80GB.json",
+        "blob_id": "dbc624731f5cb9afcdc9213183d00d1e5edd4a00"
+      },
+      {
+        "filename": "build/torch26-cxx98-cu118-x86_64-linux/moe/configs/E=8,N=4096,device_name=NVIDIA_H100_80GB_HBM3,dtype=fp8_w8a8.json",
+        "blob_id": "cc614e635ea57327c610ce79e99ae5339614f22e"
+      },
+ 
```

Comment:
Holy guacamole. Thatś a long file....

Reply:
I think we can forego the per-file hashes later?